modules/mono/csharp_script.cpp

```diff
@@ -2246,6 +2246,17 @@ bool CSharpScript::_update_exports(PlaceHolderScriptInstance *p_instance_to_upda
 			} else {
 				p_instance_to_update->update(propnames, values);
 			}
+		} else if (placeholders.size()) {
+			uint64_t script_modified_time = FileAccess::get_modified_time(get_path());
+			uint64_t last_valid_build_time = CSharpLanguage::get_singleton()->gdmono->get_project_assembly_modified_time();
```

Comment:
You could use `GDMono::get_singleton` to obtain it directly.

```suggestion
			uint64_t last_valid_build_time = GDMono::get_singleton()->get_project_assembly_modified_time();
```